var/spack/repos/builtin/packages/py-bluepy/package.py

```diff
@@ -80,3 +80,14 @@ def patch(self):
         filter_file("'jsonschema>=2.3.0',", "", "setup.py")
         filter_file("'progressbar2>=3.18',", "", "setup.py")
         filter_file("'shapely>=1.3.2',", "", "setup.py")
+
+    @property
+    def import_modules(self):
+        if self.version < Version('2.0.0'):
```

Comment:
I think this is fine, although I was wondering if we could `super().import_modules` and then filter the result; currently if the anything changes in bluepy, we have to remember to add new modules to this list.